shared/scripts/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@aragon/apps-shared-scripts",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "echo \"No tests\""
+  }
```

Comment:
At this point, I think we can keep this package `private` until we absolutely need to make it public (e.g. for the apps to be `npm install`ed correctly).

Reply:
done